Server/sprawl_calculator/urban_sprawl/clip_raster/raster_downloader.py

```diff
@@ -0,0 +1,8 @@
+import requests
+from tempfile import NamedTemporaryFile
+
+def download_raster(url):
+    response = requests.get(url)
+    with NamedTemporaryFile(delete=False, suffix='.tif') as f:
```

Comment:
Two comments: we should check the status_code response to request 200 only then we can save it so add a status code check , the url type is string so you can put a `url:str` for typing help.